src/openvic-simulation/military/UnitType.cpp

```diff
@@ -1,5 +1,6 @@
 #include "UnitType.hpp"
 
+#include "openvic-simulation/dataloader/NodeTools.hpp"
 #include "openvic-simulation/country/CountryInstance.hpp"
```

Comment:
```suggestion
#include "openvic-simulation/country/CountryInstance.hpp"
#include "openvic-simulation/dataloader/NodeTools.hpp"
```
Include formatting (alphabetical order)